pennylane/qchem/tapering.py

```diff
@@ -216,40 +216,6 @@ def paulix_ops(generators, num_qubits):
     return paulixops
 
 
-def _observable_mult(obs_a, obs_b):
```

Comment:
Should we keep this (maybe deprecate) for some time before deleting it?

Reply:
We could. I assumed that it's a private method so only we should be the ones using it. I think its fine to remove out-right